osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/Event.java

```diff
@@ -10,32 +10,53 @@
 
 import java.io.Serializable;
 
+import lombok.Getter;
 import org.joda.time.DateTime;
 
+@Getter
 public class Event implements Serializable {
 
-    /**
-     * Serial Version UID.
-     */
     private static final long serialVersionUID = 4482313912422705642L;
     private final DateTime timestamp;
     private final Integer eventCode;
-    private final Integer eventCounter;
     private final EventLogCategory eventLogCategory;
+    private final Integer eventCounter;
+    private final DateTime startTime;
+    private final Long duration;
 
     public Event(final DateTime timestamp, final Integer eventCode, final Integer eventCounter,
             final EventLogCategory eventLogCategory) {
         this.timestamp = timestamp;
         this.eventCode = eventCode;
         this.eventCounter = eventCounter;
         this.eventLogCategory = eventLogCategory;
+        this.startTime = null;
+        this.duration = null;
```

Comment:
Replace this code with `this(...)`

Reply:
I removed the extra constructor and have agreed to calling the constructor with null-values.